src/main/java/subsystems/Leds/LedConstants.java

```diff
@@ -0,0 +1,5 @@
+package subsystems.Leds;
+
+public final class LedConstants {
```

Comment:
This class is inside the `Leds` package, so calling it `Constants` is enough. Don't forget the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.